### PR TITLE
Refactor/activity and test

### PR DESCRIPTION
**Pull request by okalimbet**
## What is the change?
New helper functions were added to follow SRP rules: getUserInfoByDateAndId() and filterUserSortedArray()
Fixed syntax in Activity.js and activity-test.js
New tests were added to test newly created helper functions

## What does it fix? 
SRP and readability

## Is this a fix or a feature? 
fix

## Where should the reviewer start?
Activity.js:
line 5 

activity-test.js:
line 204

style.scss: 
line 61, 150, 163 - variable names were changed

## How should this be tested?
Running npm test passes all tests.

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -2,90 +2,117 @@ class Activity {
   constructor(activityData) {
     this.activityData = activityData
   }
+
+  getUserInfoByDateAndId(id, date) {
```
Comment (mdflynn) [inline src/Activity.js:6]: Good job pulling this out and creating it's own method.

File: src/Activity.js [hunk 2]
```diff
@@ -2,90 +2,117 @@ class Activity {
...
-    return userRepo.combineRankedUserIDsAndAveragedData(friendsActivity, date, 'numSteps', timeline)
+    return userRepo.combineRankedUserIDsAndAveragedData(friendsActivity, date, 'numSteps', timeline);
   }
+
   showChallengeListAndWinner(user, date, userRepo) {
     let rankedList = this.getFriendsAverageStepsForWeek(user, date, userRepo);
 
-    return rankedList.map(function(listItem) {
-      let userID = Object.keys(listItem)[0];
+    return rankedList.map(stepsById => {
```
Comment (mdflynn) [inline src/Activity.js:81]: Nice job renaming variables. Easier to read and understand now.

File: src/Activity.js (hunk 1 above)
Comment (cameronRomo) [inline src/Activity.js:6]: Strong SRP in this file now!

**Review by mdflynn — APPROVED** [review]
This is a nice job refactoring the class and tests.

**Review by cameronRomo — COMMENTED** [review]
This refactor is helped SRP a lot!

**Pull request merged**